lms/services/vitalsource/_client.py

```diff
@@ -103,10 +214,12 @@ class Meta:
             unknown = EXCLUDE
 
         title = fields.Str(required=True)
+        """Title of the chapter."""
+
         cfi = fields.Str(required=True)
-        page = fields.Str(required=True)
+        """A reference to the location within the book."""
 
-        url = fields.Str(required=False)
-        """vitalsource:// like url identifying the book and chapter"""
+        page = fields.Str(required=True)
+        """The start page of the chapter."""
```

Comment:
The URL param has gone here because it should have gone before. We have this as "not required" but in truth it never comes from the API, we add it on ourselves later, so this does nothing.